build/ci/run-e2e-features-enabled-tests.yml

```diff
@@ -26,7 +26,8 @@ jobs:
         }
 
   - bash: |
-      echo "##vso[task.setvariable variable=Resource]$(testFeaturesEnabledEnvironmentUrl)"
+      echo "##vso[task.setvariable variable=testEnvironmentUrl]$(testServerDataPartitionEnabledUrl)"
```

Comment:
The variable and the file name doesnt match. Should we keep it consistent. One says dataparition and other says feature-enabled

Reply:
Done. Changed it back to FeaturesEnabled